scrapy/http/response/text.py

```diff
@@ -68,6 +72,15 @@ def body_as_unicode(self):
                       ScrapyDeprecationWarning)
         return self.text
 
+    def json(self):
+        """
+        Deserialize a JSON document to a Python object.
+        """
+        if self._cached_decoded_json is _NONE:
+            self._cached_decoded_json = json.loads(self.text)
+
```

Comment:
I'd remove this line

Reply:
@elacuesta thank you, good catch! Fixed. If possible, can we merge it?